common/djangoapps/util/milestones_helpers.py

```diff
@@ -345,13 +345,15 @@ def add_course_content_milestone(course_id, content_id, relationship, milestone)
     return milestones_api.add_course_content_milestone(course_id, content_id, relationship, milestone)
 
 
-def get_course_content_milestones(course_id, content_id, relationship):
+def get_course_content_milestones(course_id, content_id, relationship, user_id=None):
     """
     Client API operation adapter/wrapper
     """
     if not settings.FEATURES.get('MILESTONES_APP', False):
         return []
     from milestones import api as milestones_api
+    if user_id:
+        return milestones_api.get_course_content_milestones(course_id, content_id, relationship, {'id': user_id})
```

Comment:
I'm not too keen on the non-DRY call here.  Instead, keep one call, but optionally pass in the value to the `user` parameter.